doc/source/reference/arrays.dtypes.rst

```diff
@@ -401,19 +425,21 @@ Type strings
 
     .. admonition:: Example
 
-       Data type with fields ``r``, ``g``, ``b``, ``a``, each being
-       an 8-bit unsigned integer:
+         >>> import numpy as np
```

Comment:
IMO these new imports should appear along with the code right before they're needed, not in a new code block preceding any text that's already there.

Reply:
Thank you, addressed in 811830b955a617ff5dddc4df5f3fdd85e94aca31 (sorry for the force-push, forgot to skip Azure Pipelines and CirrusCI).